src/components/comment-card.jsx

```diff
@@ -0,0 +1,32 @@
+// import {useState, useEffect} from 'react';
+import '../test.css';
+
+
+
+function CommentCard({comment}) {
+    console.log(comment)
+    return (
+        <div>
+            <div className='topicandposter'>                       
+                    
+                <p className='vtopicandposterelements'>posted by: {comment.author} at {comment.created_at}</p>
+            </div>
```

Comment:
Here we would want to add some extra logic, so if there are no comments yet on a post, it will render a message such as 'No comments yet' 

😄